src/main/java/net/logstash/logback/appender/DelegatingAsyncDisruptorAppender.java

```diff
@@ -49,29 +54,69 @@
     private final AppenderAttachableImpl<Event> appenders = new AppenderAttachableImpl<Event>();
     
     private class DelegatingEventHandler implements EventHandler<LogEvent<Event>> {
+        /**
+         * Whether exceptions should be reported with a error status or not.
+         */
+    	private boolean silentError;
```

Comment:
Looks like your editor is configured to use tabs.  Please convert your additions to use spaces to keep consistent with the rest of the file.

Reply:
Indeed..